docs/reST/ref/mouse.rst

```diff
@@ -58,13 +60,14 @@ to access data about the mouse scroll, such as ``which`` (it will tell you what
    
    def main():
       while True:
-         for event_var in pygame.event.get():
-               if event_var.type == QUIT:
+         for event in pygame.event.get():
+               if event.type == QUIT:
                   pygame.quit()
                   return
-               elif event_var.type == MOUSEWHEEL:
-                  print(event_var) # can access properties with prop notation 
-                                   # (ex: event_var.y)
+               elif event.type == MOUSEWHEEL:
+                  print(event) 
```

Comment:
Perhaps printing the mouse wheel attributes would be more instructive for the reader.
`print(event.x, event.y, event.flipped, event.which)`

When I went to look up the event MOUSEWHEEL event on the event page of the documentation I didn't see the attributes listed.

From event.c I see these attributes (flipped, x, y, which).
```c
        case SDL_MOUSEWHEEL:
            /* https://wiki.libsdl.org/SDL_MouseWheelEvent */
#ifndef NO_SDL_MOUSEWHEEL_FLIPPED
            _pg_insobj(dict, "flipped", PyBool_FromLong(event->wheel.direction == SDL_MOUSEWHEEL_FLIPPED));
#else
            _pg_insobj(dict, "flipped", PyBool_FromLong(0));
#endif
            _pg_insobj(dict, "y", PyInt_FromLong(event->wheel.y));
            _pg_insobj(dict, "x", PyInt_FromLong(event->wheel.x));
            _pg_insobj(dict, "which", PyInt_FromLong(event->wheel.which));
            break;
```

Reply:
Done :)